### PR TITLE
[MM-57962] Remove outdated Copy Link menu in favour of native Electron dropdown

### PR DESCRIPTION
**Pull request by devinbinnie**
#### Summary
We noticed that the tooltip on the team sidebar was hovering over top of the Copy URL context menu built into the web app for use in the Desktop App. Upon closer investigation, it was determined that this menu was no longer needed and could be replaced functionally identically with the built-in Electron context menu, so this PR removes that component entirely.

I had to also add `user-select: none` to the buttons that had this menu on since apparently right-clicking would select the text. In the screenshot, you may also notice that on Windows/Linux there is a `Select All` function now. This just selects the whole page, and is pretty harmless, albeit a bit in the way.

Tested on macOS (latest nightly ~v5.8), Windows 10 (Desktop v5.7) and Ubuntu 22.04 (Desktop v5.6) - all working as expected.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-57962

#### Screenshots
|  before  |  after  |
|----|----|
| image | image |

```release-note
NONE
```

### CODE REVIEW

**Review by harshilsharma63 — APPROVED** [review]

**Comment by harshilsharma63** [comment]
White the code is fine, I must say the new menu looks very ugly and is a step backward in terms of UI aesthetics. cc @abhijit-singh 

I feel the before version is significantly better.

**Review by yasserfaraazkhan — APPROVED** [review]
image

the copy link button is accessible.

File: webapp/channels/src/i18n/fa.json [hunk 1]
```diff
@@ -2684,7 +2684,6 @@
   "copied.message": "رونوشت شد",
   "copy.code.message": "رونوشت کد",
   "copy.text.message": "رونوشت متن",
-  "copy_url_context_menu.getChannelLink": "لینک را کپی کنید",
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/i18n/fa.json:2687]: i dont think we should be doing any addition/deletion in the other languages file. Can you revert these

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

**Comment by devinbinnie** [comment]
> White the code is fine, I must say the new menu looks very ugly in terms of UI aesthetics. cc @abhijit-singh 
> 
> I feel the before version is significantly better.

@harshilsharma63 If you're referring to the screenshot, that's from my Ubuntu VM, it will now use whatever the native menu is. I was using Ubuntu at the time to test that it worked.

If you pop open the server on your local Desktop App you'll see the Mac version which is better looking.

File: webapp/channels/src/i18n/fa.json (hunk 1 above)
Comment (devinbinnie) [reply webapp/channels/src/i18n/fa.json:2687]: I wasn't sure about this, but I do know that we don't need these strings anymore. Are we sure these will be cleaned up in weblate later?

Comment (devinbinnie) [reply webapp/channels/src/i18n/fa.json:2687]: I guess CI answered that for me :P

**Review by abhijit-singh — APPROVED** [review]
Looks good!

**Review by M-ZubairAhmed — APPROVED** [review]

**Pull request merged**